app/services/postcard.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var Config = require('../../config');
+
+module.exports = function (API) {
+  var Postcard = {};
```

Comment:
nit: we usually pad our services, controllers, and states with new lines, so there should be a new line before this line and one after line 14